pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -82,7 +83,7 @@ func (hm *hostportManager) Add(id string, podPortMapping *PodPortMapping, natInt
 		return nil
 	}
 
-	if podPortMapping.IP.To4() == nil {
+	if podPortMapping.IP.To4() == nil && podPortMapping.IP.To16() == nil {
```

Comment:
should we use the net utils for this?

Reply:
which function do you suggest?

Reply:
you're right, we don't seem to have one.